src/server/endpoints/covidcast_utils/db_signals.csv

```diff
@@ -1454,65 +1454,65 @@ jhu-csse,deaths_cumulative_num,TRUE,deaths_incidence_prop,FALSE,"Confirmed COVID
+National provisional death counts is based on death certificate data received and coded by the National Center for Health Statistics ([NCHS](https://www.cdc.gov/nchs/nvss/vsrr/COVID19/index.htm))",NCHS Mortality Data,n/a,Deaths,USA,"state,nation",,2020-05,"Start dates vary by geo: nation 2020-06, state 2020-05",Ongoing,NA,week,Week,weekly,11-17 days,Weekly. All-cause mortality takes ~6 weeks on average to achieve 99% of its final value (https://link.springer.com/article/10.1057/s41271-021-00309-7),All,None,dead,"Unavailable by NCHS when counts are between 1 and 9, and for weeks where the counts are less than 50% of the expected number, since these provisional counts are highly incomplete and potentially misleading",Data is available for all states and some territories.,Value,count,late,bad,FALSE,FALSE,FALSE,FALSE,FALSE,public,public,NCHS Data Use Agreement (https://www.cdc.gov/nchs/data_access/restrictions.htm),See license,https://www.cdc.gov/nchs/data_access/restrictions.htm,https://cmu-delphi.github.io/delphi-epidata/api/covidcast-signals/nchs-mortality.html
 nchs-mortality,deaths_allcause_incidence_num,TRUE,deaths_allcause_incidence_prop,FALSE,"All Causes Deaths (Weekly new, per 100k people)",TRUE,"Number of weekly new deaths from all causes, per 100k people","Number of weekly new deaths from all causes, per 100k people.
 
-National provisional death counts is based on death certificate data received and coded by the National Center for Health Statistics ([NCHS](https://www.cdc.gov/nchs/nvss/vsrr/COVID19/index.htm))",NCHS Mortality Data,n/a,Deaths,USA,"state,nation",nation,2020-05,"Start dates vary by geo: nation 2020-06, state 2020-05",Ongoing,NA,week,Week,weekly,11-17 days,Weekly. All-cause mortality takes ~6 weeks on average to achieve 99% of its final value (https://link.springer.com/article/10.1057/s41271-021-00309-7),All,None,dead,"Unavailable by NCHS when counts are between 1 and 9, and for weeks where the counts are less than 50% of the expected number, since these provisional counts are highly incomplete and potentially misleading",Data is available for all states and some territories.,Value,per100k,late,bad,FALSE,FALSE,FALSE,FALSE,FALSE,public,public,NCHS Data Use Agreement (https://www.cdc.gov/nchs/data_access/restrictions.htm),See license,https://www.cdc.gov/nchs/data_access/restrictions.htm,
```

Comment:
Change for nchs-mortality signals removes "nation" from "Delphi-Aggregated Geography" column. These are reported as-is from the source.